config/distscript.pl

```diff
@@ -62,17 +62,20 @@ sub subst {
         if ($_ =~ m/^([^?! ].|.[^?! ]) (.+)$/) {
             my $file = $2;
             print "*** WARNING: found modified file in source tree: $file\n";
+	    my $file_ = $file;
+            $file_ =~ s/\./\_/g;
 
-            # There is one exception that is allowed: the nightly
+            # There are some exceptions allowed: the nightly
             # tarball script changes configure.ac to set the correct
             # version number.  In this case, the nightly tarball
             # script will set a magic environment variable with the
             # SHA1 hash of the ok-to-be-modified file.  See if it is
             # set, and if the SHA1 hash agrees.
-            if (exists($ENV{"LIBFABRIC_DISTSCRIPT_SHA1_$file"})) {
+            my $envsha = $ENV{"LIBFABRIC_DISTSCRIPT_SHA1_$file"} || $ENV{"LIBFABRIC_DISTSCRIPT_SHA1_$file_"};
+            if ($envsha) {
```

Comment:
I do not think these 2 new lines are correct.  If $file is modified, then the original line 72 is correct (i.e., you need the exists check).